app/Http/Controllers/User/MyRequestController.php

```diff
@@ -81,6 +84,20 @@ public function update(Request $request, $id)
             $this->notification->store($data);
         }
 
+        if ($request->status == 'returning') {
+            $book_id = $this->bookUser->findId($id)->book_id;
+            $count_returnedBook = $this->bookUser->countReturned($book_id);
+            $find_Returned = $this->bookMeta->findReturned($book_id);
+            if (empty($find_Returned)) {
+                $a = $this->bookMeta->insertReturned([
+                    'key' => 'count_returned',
+                    'value' => $count_returnedBook->book_returned,
+                    'book_id' =>$book_id,
+                ]);
+            }
+            else $this->bookMeta->updateReturned($book_id, ['value' => $count_returnedBook->book_returned]);
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Variables.ValidVariableName.NotCamelCaps</br>**Severity**: error</br>**File**: app/Http/Controllers/User/MyRequestController.php</br>**Line**: 98</br>Variable "book_id" is not in valid camel caps format